Dfe.Academies.Academisation.Domain/ProjectAggregate/Project.cs

```diff
@@ -68,12 +69,77 @@ public static CreateResult<IProject> Create(IApplication application, Establishm
 			//CapitalCarryForwardAtEndMarchNextYear = school.SchoolNFYCapitalForward.ConvertDeficitAmountToNegativeValue(school.SchoolNFYCapitalIsDeficit),
 			YearOneProjectedPupilNumbers = school.ProjectedPupilNumbersYear1,
 			YearTwoProjectedPupilNumbers = school.ProjectedPupilNumbersYear2,
-			YearThreeProjectedPupilNumbers	= school.ProjectedPupilNumbersYear3
+			YearThreeProjectedPupilNumbers = school.ProjectedPupilNumbersYear3
 		};
 
 		return new CreateSuccessResult<IProject>(new Project(projectDetails));
 	}
 
+	public CommandResult UpdatePatch(ProjectDetails detailsToUpdate)
```

Comment:
Could move this patch logic into the Service Layer to keep the domain layer clean with a conventional Update method. It's going to be more tricky to validate the patch parameter. I should've thought of that earlier.